Parsers/ASimAuditEvent/Parsers/ASimAuditEventCrowdStrikeFalcon.yaml

```diff
@@ -0,0 +1,158 @@
+      Activity: string,
+      Operation: string,
+      EventType_lookup: string,
+      EventSubType: string,
+      Object: string,
+      ObjectType: string
+  ) 
+      [
+      "delete_report_execution", "Delete Report Execution", "Delete", "", "Report Execution", "Scheduled Task",
+      "delete_scheduled_report", "Delete Scheduled Report", "Delete", "", "Scheduled Report", "Scheduled Task",
+      "update_scheduled_report", "Update Scheduled Report", "Set", "", "Scheduled Report", "Scheduled Task",
+      "create_scheduled_report", "Create Scheduled Report", "Create", "", "Scheduled Report", "Scheduled Task",
+      "update_class_action", "Update Class Action", "Set", "", "Class Action", "Other",
+      "update_policy", "Update Policy", "Set", "", "Policy", "Policy Rule",
+      "enable_policy", "Enable Policy", "Enable", "", "Policy", "Policy Rule",
+      "create_policy", "Create Policy", "Create", "", "Policy", "Policy Rule",
+      "remove_rule_group", "Remove Rule Group", "Other", "Remove", "Rule Group", "Service",
+      "create_rule_group", "Create Rule Group", "Create", "", "Rule Group", "Service",
+      "delete_rule_group", "Delete Rule Group", "Delete", "", "Rule Group", "Service",
+      "add_rule_group", "Add Rule Group", "Other", "Add", "Rule Group", "Service",
+      "delete_rule", "Delete Rule", "Delete", "", "Rule", "Policy Rule",
+      "update_rule", "Update Rule", "Set", "", "Rule", "Policy Rule",
+      "create_rule", "Create Rule", "Create", "", "Rule", "Policy Rule",
+      "disable_policy", "Disable Policy", "Disable", "", "Policy", "Policy Rule",
+      "delete_policy", "Delete Policy", "Delete", "", "Policy", "Policy Rule",
+      "update_priority", "Update Priority", "Set", "", "Policy", "Policy Rule",
+      "assign_policy", "Assign Policy", "Other", "Assign", "Policy", "Policy Rule",
+      "remove_policy", "Remove Policy", "Other", "Remove", "Policy", "Policy Rule",
+      "ip_rules_added", "IP Rules Added", "Create", "", "Rule", "Other",
+      "ip_rules_removed", "IP Rules Removed", "Delete", "", "Rule", "Other",
+      "CreateAPIClient", "Create API Client", "Create", "", "API Client", "Service",
+      "UpdateAPIClient", "Update API Client", "Set", "", "API Client", "Service"
+  ];
+  let EventSeverityLookup = datatable (LogSeverity: string, EventSeverity: string)
+      [
+      "0", "Informational",
+      "1", "Informational",
+      "2", "Low",
+      "3", "Medium",
+      "4", "High",
+      "5", "High"
+  ];
+  let UserAuditActivities = dynamic(["delete_report_execution", "delete_scheduled_report", "update_scheduled_report", "create_scheduled_report", "update_class_action", "update_policy", "enable_policy", "create_policy", "remove_rule_group", "create_rule_group", "delete_rule_group", "add_rule_group", "delete_rule", "update_rule", "create_rule", "disable_policy", "delete_policy", "update_priority", "assign_policy", "remove_policy", "ip_rules_added", "ip_rules_removed"]);
```

Comment:
@niralishah-crest @jayeshprajapaticrest 
I found additional Activity values under UserActivityAuditEvent classid in the sample I had. Would you be able to validate these if they can included as well 'Audit'?
["detection_update","quarantined_file_update","update_group","hide_host_requested","create_report","selector_hit_update","downloadSensorInstaller","add_group","reveal_token","new_exception","lift_containment_requested","create_group","containment_requested","update","activate","add_exclusion","delete_report","add_ss_ioa_exclusion","create_report_execution","delete_group","remove_group","delete_exception","remove_ss_ioa_exclusion","update_filter","override","update_ss_ioa_exclusion","unhide_host_requested","update_exclusion","remove_exclusion","remove_sv_exclusion","mobile_hide_host_requested","update_rule_group","create_filter"]

Reply:
We have gone through the above list of activities but we are unable to find many of them in our live environment as well as in document like update_ss_ioa_exclusion, remove_ss_ioa_exclusion, add_ss_ioa_exclusion, create_filter, downloadSensorInstaller etc. 

Also there are some activities like add_group, delete_group, create_group which are more related to UserManagement but as we discussed earlier due to limited  fields are mentioned in configuration file, we are unable to map it with UserManagement schema. As these are group related activities, we can't consider them as Audit Activities as you mentioned in SentinelOne.
Here, I'm attaching the document.

[Streaming API Event Dictionary _ Tools and Reference _ Documentation _ Support and resources _ Falcon.pdf](https://github.com/Azure/Azure-Sentinel/files/13486052/Streaming.API.Event.Dictionary._.Tools.and.Reference._.Documentation._.Support.and.resources._.Falcon.pdf)

Reply:
ok thanks @niralishah-crest @jayeshprajapaticrest